src/mmw/js/src/modeling/controls.js

```diff
@@ -12,6 +12,32 @@ var $ = require('jquery'),
     conservationPracticeTmpl = require('./templates/controls/conservationPractice.html'),
     precipitationTmpl = require('./templates/controls/precipitation.html');
 
+function getHumanReadableLabel(value) {
```

Comment:
I kind of feel as thought this helper should be part of the ModificationModel or at least in the models file.